### PR TITLE
Add doc open suffix option

### PR DESCRIPTION
**Pull request by sagivf**
- bump language service version
- add doc suffix support

### CODE REVIEW

File: package/src/languageFeatures.ts [hunk 1]
```diff
@@ -886,7 +886,7 @@ export class CompletionAdapter implements monaco.languages.CompletionItemProvide
                     const fullURL = linkWithoutPostfix.startsWith('https')
                         ? linkWithoutPostfix
                         : `${documentationBaseUrl}/${linkWithoutPostfix}`;
-                    return monaco.Uri.parse(fullURL);
+                    return monaco.Uri.parse(`${fullURL}${documentationSuffix || ''}`);
```
Comment (maxburs) [inline package/src/languageFeatures.ts:889]: IMO this function is a little hard to read right now. I'd write it more like this:
```ts
get(_target, prop, _receiver) {
    // The link comes with a postfix of ".md" that we want to remove
    let url = prop.toString().replace('.md', '');
    // Sometimes we get the link as a full URL. For example in the main doc link of the item
    if (url.startsWith('https')) {
        url = `${documentationBaseUrl}/${url}`;
    }
    if (documentationSuffix) {
        url += documentationSuffix;
    }
    return monaco.Uri.parse(url);
}
```

**Review by maxburs — APPROVED** [review]

File: package/src/languageService/settings.ts [hunk 2]
```diff
@@ -18,6 +18,7 @@ export interface LanguageSettings {
     quickFixCodeActions?: QuickFixCodeActionOptions[];
     enableQuickFixes?: boolean;
     documentationBaseUrl?: boolean;
+    documentationSuffix?: string;
```
Comment (maxburs) [inline package/src/languageService/settings.ts:21]: Might be worth adding a doc comment here

Comment (sagivf) [reply package/src/languageService/settings.ts:21]: Where exactly? Knowhere else has

Comment (maxburs) [reply package/src/languageService/settings.ts:21]:
```suggestion
    /**
     *  Enables god mode
     */
    documentationSuffix?: string;
```

**Review by maxburs — APPROVED** [review]

**Review by maxburs — APPROVED** [review]

**Pull request merged**